.gitlab-ci.yml

```diff
@@ -1,8 +1,96 @@
 stages:
   - test
+  - deploy
+
+variables:
+  NVM_VERSION: 0.39.1
+  NODE_VERSION: 10.19.0
 
 test:
   stage: test
   image: ubuntu:20.04
+  before_script:
+    - apt-get update
+    - apt-get upgrade -y
+    - apt-get install -y tzdata
+    - apt-get install -y wget
+    - apt-get install -y git
+    - git submodule update --init
+    - wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v$NVM_VERSION/install.sh | bash
+    - export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")" && [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+    - source ~/.bashrc
```

Comment:
What do we get from sourcing the `.bashrc` ?

Reply:
This can be removed. I don't think this is required now.
Earlier I was adding the nvm command in a different way, which required sourcing the **bashrc**, but that didn't work and I had to do it differently.